javascript-modal/styles.css

```diff
@@ -0,0 +1,77 @@
+* {
+  box-sizing: border-box;
+}
+
+#trigger {
+  width: 12vw;
+  height: 5vh;
+  border: none;
+  border-radius: 7px;
+  font-size: 1.5rem;
+  color: #fff;
+  background-color: #4a4ae9;
+  box-shadow: 0 2px 4px black;
+}
+
+#trigger:hover {
+  cursor: pointer;
+}
+
+#trigger:focus {
+  display: none;
+}
```

Comment:
You should not need this. See if you can make it work without including this rule.